backend/app.env

```diff
@@ -3,4 +3,5 @@ DB_SOURCE=postgresql://root:secret@localhost:5432/good_grocer?sslmode=disable
 SERVER_ADDRESS=0.0.0.0:8080
 TOKEN_SYMMETRIC_KEY=01234567890123456789012345678901
 ACCESS_TOKEN_DURATION=15m
-GOOGLE_MAPS_API_KEY=AIzaSyA1nWQO2sDr2MVzipOLMvApGd3cGWUgc1s
\ No newline at end of file
+GOOGLE_MAPS_API_KEY=AIzaSyA1nWQO2sDr2MVzipOLMvApGd3cGWUgc1s
+ENV=dev
```

Comment:
is this going to be a problem for the workflow or will env be set to prod somewhere else for that?

Reply:
yeah ive set it to prod on AWS